tests/framework/microvm.py

```diff
@@ -409,6 +410,14 @@ def get_all_metrics(self):
         """Return all metric data points written by FC."""
         return [json.loads(line) for line in self.metrics_file.read_text().splitlines()]
 
+    def flush_metrics_to_cw(self, metrics):
+        """Uploads all metric data points emitted by Firecracker to CloudWatch."""
+
+        self.api.actions.put(action_type="FlushMetrics")
+        fc_metrics = open(self.metrics_file, encoding="utf-8").read().replace("\n", "")
+        fc_metrics = fc_metrics.replace(" ", "").replace("}{", "}\n{")
+        flush_fc_metrics_to_cw(fc_metrics, metrics)
```

Comment:
>It doesn't need to know that the metrics are getting sent to CloudWatch.

I get this part but I think instead of polling every X secs it would be simpler to dump all metrics at the end of each test. I'll move the fc metrics related stuff from metrics.py to a new file and add something to discuss.